doc/release-process.md

```diff
@@ -28,10 +28,10 @@ Release Process
 
 * Update hardcoded [seeds](/contrib/seeds/README.md), see [this pull request](https://github.com/bitcoin/bitcoin/pull/7415) for an example.
 * Update [`src/chainparams.cpp`](/src/chainparams.cpp) m_assumed_blockchain_size and m_assumed_chain_state_size with the current size plus some overhead (see [this](#how-to-calculate-m_assumed_blockchain_size-and-m_assumed_chain_state_size) for information on how to calculate them).
-* Update `src/chainparams.cpp` chainTxData with statistics about the transaction count and rate. Use the output of the RPC `getchaintxstats`, see
-  [this pull request](https://github.com/bitcoin/bitcoin/pull/17002) for an example. Reviewers can verify the results by running `getchaintxstats <window_block_count> <window_last_block_hash>` with the `window_block_count` and `window_last_block_hash` from your output.
-* Update `src/chainparams.cpp` nMinimumChainWork with information from the getblockchaininfo rpc.
-* Update `src/chainparams.cpp` defaultAssumeValid with information from the getblockhash rpc.
+* Update `src/chainparams.cpp` chainTxData with statistics about the transaction count and rate. Use the output of the `getchaintxstats` RPC, see
+  [this pull request](https://github.com/bitcoin/bitcoin/pull/17002) for an example. Reviewers can verify the results by running `getchaintxstats <window_block_count> <window_final_block_hash>` with the `window_block_count` and `window_final_block_hash` from your output.
+* Update `src/chainparams.cpp` nMinimumChainWork with information from the `getblockchaininfo` RPC.
+* Update `src/chainparams.cpp` defaultAssumeValid with information from the `getblockhash` RPC.
```

Comment:
Should both lines 33 and 34 refer to `getblockheader` instead? e.g.:

* Update `src/chainparams.cpp` nMinimumChainWork, defaultAssumeValid, and the block height comment with information from the `getblockhash` RPC.

Reply:
Good idea! Particularly I think `getblockchaininfo` should not be used here, as it only shows the chainwork for the _last_ block, i.e. it is not useful for potential reviewers of the updated parameters. Maybe refer to `getblockheader` RPC but also add a note to `getblockhash` in parantheses, as this is useful to convert block height to block hash? (As pointed out in https://github.com/bitcoin/bitcoin/pull/20263#pullrequestreview-521252569, starting with the block height for both updating and reviewing the chain parameter updates would feel most natural to me).

Reply:
Oops, I meant to write getblockheader instead of getblockhash, but you're right to mention both and I like how you did your review of 20263. Done (thanks!)